app/src/main/res/values/integers.xml

```diff
@@ -2,4 +2,7 @@
 <resources>
     <integer name="default_anim_length">250</integer>
     <integer name="shorter_anim_length">150</integer>
+    <integer name="default_number_of_names_per_group">2</integer>
+    <integer name="default_number_of_groups">1</integer>
+
```

Comment:
sorry for this empty line I've deleted it and will be pushed in the next commit.